pages/transactions.js

```diff
@@ -106,4 +213,15 @@ class TransactionsPage extends React.Component {
   }
 }
 
-export default withUser(addCollectiveCoverData(TransactionsPage));
+const addTransactionsData = graphql(transactionsQuery, {
+  skip: props => !props.slug,
+  name: 'transactionsQuery',
```

Comment:
This is not a good name, it's confusing.

Reply:
I agree that it creates confusion with the query itself, something like `transactionsQueryResult` would make more sense to me.

Reply:
I renamed it to `transactionData`.

Reply:
Looks better!